packages/pipeline/src/entities/token_metadata.ts

```diff
@@ -14,3 +14,9 @@ export class TokenOnChainMetadata {
     @Column({ type: 'nvarchar', nullable: false })
     public name!: string;
 }
+
+@Entity({ name: 'trusted_tokens', schema: 'raw' })
```

Comment:
I imagine it is common for different authorities to use different names for the same token (e.g. "0x", "ZeroEx", "0x Protocol", etc.). Our schema should allow for this, but it looks like it currently does not because `address` is the only primary column for `TokenOnChainMetadata`.

What do you think about removing the `TrustedTokens` entity and simply including `authority` as the second primary column on `TokenOnChainMetadata`?

So `TokenOnChainMetadata` would look like this:

```ts
export class TokenOnChainMetadata {
    @PrimaryColumn()
    public address!: string;

    @PrimaryColumn()
    public authority!: string;

    // ...
}
```

Reply:
The reason I did a separate table was to reduce dependencies in the raw layer esp since I envision token_on_chain_metadata coming from the blockchain and trusted sources coming from other providers. 

The thing you're describing, however makes a lot of sense to have in the "concepts" layer (the curated core analytics building blocks). What do you think of producing the combined table there instead?